files/lib/system/discord/DiscordApi.class.php

```diff
@@ -1218,9 +1218,13 @@ public function createGuild($params)
      *
      * @return  array
      */
-    public function getGuild()
-    {
+    public function getGuild(bool $withCounts = false)
+    {        
```

Comment:
why here are new spaces? ^^ please remove the spaces, because they are not psr-12 compatible :)